### PR TITLE
[MAIN] Add file header. Remove accel from Ok and Cancel buttons in all languages. Add missing accels.

### PR DESCRIPTION
**Pull request by zecarlos1957**
Add file header
Remove accel from Ok and Cancel buttons in all languages
Add missing accelerators and fix some accel collisions
Make IDOK button a DEFPUSHBUTTON

### CODE REVIEW

**Comment by zecarlos1957** [comment]
VirtualBox_WinSrv2003_Lock_Ok_Cancel
VirtualBox_WinSrv2003_Ok_Cancel

File: dll/cpl/main/lang/pt-PT.rc [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Main Control Panel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```
Comment (binarymaster) [inline dll/cpl/main/lang/pt-PT.rc:5]:
```suggestion
 * TRANSLATOR:  Copyright 2019-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```

File: dll/cpl/main/lang/en-US.rc [hunk 2]
```diff
@@ -40,8 +40,8 @@ BEGIN
     LTEXT "Short", -1, 50, 35, 27, 10
     LTEXT "Long", -1, 157, 35, 27, 10
     CONTROL "", IDC_SLIDER_CLICK_LOCK, "msctls_trackbar32", TBS_AUTOTICKS | WS_TABSTOP, 78, 35, 70, 17
-    PUSHBUTTON "&OK", IDOK, 110, 59, 60, 14
-    PUSHBUTTON "&Cancel", IDCANCEL, 174, 59, 60, 14
+    PUSHBUTTON "OK", IDOK, 110, 59, 60, 14
```
Comment (binarymaster) [inline dll/cpl/main/lang/en-US.rc:43]: What about making OK button a `DEFPUSHBUTTON`?

Comment (zecarlos1957) [reply dll/cpl/main/lang/en-US.rc]: Right. Thanks.

**Review by JoachimHenze — APPROVED** [review]
Approved. Will be merged in a week. Please don't add more to it, or it will become harder to review it, especially the pt-PT-part. I checked: The changes to that currently did not introduce any new collision.

**Comment by binarymaster** [comment]
@JoachimHenze I've separated the two logically different changes into two commits.

When merging, please use "Rebase and merge" strategy.

**Review by binarymaster — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

**Pull request merged**